crates/host/src/wasmbus/mod.rs

```diff
@@ -1323,14 +1316,14 @@ pub struct Host {
     stop_rx: watch::Receiver<Option<Instant>>,
     queue: AbortHandle,
     // Component ID -> All Links
-    links: RwLock<HashMap<String, HashSet<InterfaceLinkDefinition>>>,
+    links: RwLock<HashMap<String, Vec<InterfaceLinkDefinition>>>,
     actor_claims: Arc<RwLock<HashMap<String, jwt::Claims<jwt::Actor>>>>, // TODO: use a single map once Claims is an enum
     provider_claims: Arc<RwLock<HashMap<String, jwt::Claims<jwt::CapabilityProvider>>>>,
     config_data_cache: Arc<RwLock<ConfigCache>>,
     metrics: Arc<HostMetrics>,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, Default)]
```

Comment:
I don't feel strongly about it, but I would not expect this struct to implement `Default` - I feel like every component specification requires a URL